sn_cli/src/subcommands/files/iterative_uploader.rs

```diff
@@ -95,7 +101,7 @@ impl IterativeUploader {
 
 ///////////////// Associative Functions /////////////////
 
-fn spawn_progress_handler(
+fn spawn_file_upload_events(
```

Comment:
This function handles the file upload events that we receive, so `handler` seems more apt here?

Reply:
i try to avoid descriptions like 'handler', its too generic

Reply:
can we just call it **spawn_file_upload_events_handler**
the name gets a little longer but this PR has been up for days and I just want to get this through

Reply:
Yes! I feel that is more descriptive!

Reply:
changes made. cheers roland